messaging/kafka-avro-reactive-messaging/src/test/java/io/quarkus/ts/openshift/messaging/kafka/StrimziKafkaResource.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.ts.openshift.messaging.kafka;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.strimzi.StrimziKafkaContainer;
+import org.jboss.logging.Logger;
+import org.testcontainers.containers.Network;
+import org.testcontainers.lifecycle.Startables;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class StrimziKafkaResource implements QuarkusTestResourceLifecycleManager {
+
+    private static final Logger LOG = Logger.getLogger(StrimziKafkaResource.class);
+    private StrimziKafkaContainer kafkaContainer;
+    private SchemaRegistryContainer schemaRegistry;
+
+    @Override
+    public Map<String, String> start() {
+        Network network = Network.newNetwork();
+
+        kafkaContainer = new StrimziKafkaContainer("0.18.0-kafka-2.5.0").withNetwork(network);
+        schemaRegistry = new SchemaRegistryContainer("apicurio/apicurio-registry-mem", "1.2.2.Final", 8080).withNetwork(network).withKafka(kafkaContainer, 9092);
+
+        Startables.deepStart(Stream.of(kafkaContainer, schemaRegistry)).join();
+
+        String kafkaUrl = kafkaContainer.getBootstrapServers();
+        String registryUrl = schemaRegistry.getSchemaRegistryUrl();
+        LOG.info(String.format("TestContainers Kafka URL -> %s", kafkaUrl));
+        LOG.info(String.format("TestContainers Registry URL -> %s", registryUrl));
+
+        Map<String, String> config = new HashMap<>();
+        config.put("kafka.bootstrap.servers", kafkaUrl);
+        config.put("mp.messaging.connector.smallrye-kafka.apicurio.registry.url", registryUrl + "/api");
+        return config;
+    }
+
+    @Override
+    public void stop() {
+        kafkaContainer.close();
```

Comment:
The schemaRegistry is not closed. I tried to run the tests several times and I ended up with a few up and running containers of apicurio-registry.